### PR TITLE
test(static-props): Improve static props proxying test cases

### PR DESCRIPTION
**Pull request by solkimicreb**
Improve static property proxying test cases to cover a few more edge cases

### CODE REVIEW

File: __tests__/staticProps.test.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import { Component } from 'react'
 import { view } from 'react-easy-state'
 
-describe('static props', () => {
-  test('view() should proxy react specific static properties from the component', () => {
+describe.only('static props', () => {
```
Comment (rolandszoke) [inline __tests__/staticProps.test.js]: .only
